docs/developerguide.rst

```diff
@@ -9,4 +9,43 @@ understand REANA source code and contribute to it.
 Debugging
 ---------
 
-We use ``wdb``.
+Assuming that REANA is being run in development mode following
+`reana-cluster documentation
+<http://reana-cluster.readthedocs.io/en/latest/developerguide.html#deploying-latest-master-branch-versions>`_,
+to debug a component we need to rebuild its Docker image in debug mode. Let say
+we are going to debug `reana-server`:
+
+.. code-block:: console
+
+   $ # if we are not connected to the minikube docker daemon we should do it
+   $ eval "$(minikube docker-env)"
+   $ docker build . --build-arg DEBUG=true -t reanahub/reana-server:v0.1.0
+
+Set the breakpoint in your current project code using your preferred editor and
+add `import wdb; wdb.set_trace()` instruction, for instance in the
+`GET /api/analyses` endpoint. Then, we kill `reana-server` pod to update the
+source code and we query the mentioned endpoint:
+
+.. code-block:: console
+
+   $ kubectl delete pod --selector=app="server"
+   $ curl "192.168.99.100:30659/api/analyses?organization=default&user=00000000-0000-0000-0000-000000000000"
+
+After doing that we can go to the `wdb` dashboard (you can get ``wdb`` address using `reana-cluster get wdb <http://reana-cluster.readthedocs.io/en/latest/cliapi.html#reana-cluster-get>`_).
+
+.. image:: /_static/wdb-active-sessions.png
+
+And finally select the debugging session.
+
+.. image:: /_static/wdb-debugging-ui.png
+
+
+Limitations
```

Comment:
Better not to introduce a new section here to clarify with the other forthcoming tips.  Perhaps **Limitations** in bold is enough.